_data/leads.yml

```diff
@@ -1,39 +1,50 @@
 
-- name: Greta Keyes
+- name: Anastasia Kaurova
   subteam: Electrical
-  image: greta.jpg
-  bio: Greta Keyes is the electrical lead at South Eugene Robotics team. She joined as soon as possible during her freshman year and initially was interested in mechanical before joining the electrical subteam. She loves teaching new SERT members about the electrical systems, cheering (really loudly) for the robot and drive team during competition, and organizing the shop. When not attempting to manage wire spaghetti or troubleshoot problems with the encoders, she enjoys learning about chemistry and biology, going on hikes, and reading and writing science fiction in her free time.
-
-- name: Tiffany Huang
+  image: Aanastasia.jpg
+  bio: Anastasia is a sophomore and the electrical lead. She has found a passion in working on the small and delicate aspects of the robot, all while sitting inside it. If recruited by mechanical, her joy in using loud power tools is only tempered by being in unfamiliar territory. When given her freedom from the foundry, Anastasia enjoys sailing, baking treats, and watching Marvel.
+  
+- name: Kylie McKeon
   subteam: Business
-  image: tiffany.jpg
-  bio: Senior Tiffany Huang is the business subteam lead at South Eugene Robotics Team. Tiffany joined the team during her sophomore year, and is always willing to help out new members and keep everyone else in line. When she’s not helping the business team get everything done on time, Tiffany enjoys dancing ballet, debating, playing piano, working with student government, and leading South’s Asian Student Union.
+  image: James.jpg
+  bio: Kylie McKeon is a junior and the business lead of the South Eugene Robotics Team. She joined the team freshman year and loved the business team immediately. Kylie loves helping the team raise money and spreading the word about robotics. In her free time outside robotics, Kylie enjoys reading, practicing circus, and helping out clubs around South Eugene High School.
 
-- name: Sander Moffit
-  subteam: Strategy
-  image: sander.jpg
-  bio: Sander Moffitt is a junior and strategy co-lead for South Eugene Robotics Team. This is her third year on SERT. She fell in love with robotics at the age of eight after building a LEGO robot named “BALL-E” that shot foam balls at her cat whenever it went near her bedroom. She has since evolved to leading robot design discussions and running match strategy. Outside of robotics, Sander enjoys editing her novel, painting, and coming up with creative ways to torment her little sister.
+- name: Keegan Gilds
+  subteam: Co-Strategy
+  image: Sander.jpg
```

Comment:
I assume this should be Keegan's image 😛

Reply:
Yes, It should be.